lib/internal/http2/core.js

```diff
@@ -354,10 +354,7 @@ function onSelectPadding(fn) {
     const frameLen = paddingBuffer[PADDING_BUF_FRAME_LENGTH];
     const maxFramePayloadLen = paddingBuffer[PADDING_BUF_MAX_PAYLOAD_LENGTH];
     paddingBuffer[PADDING_BUF_RETURN_VALUE] =
-        Math.min(maxFramePayloadLen,
-                 Math.max(frameLen,
-                          fn(frameLen,
-                             maxFramePayloadLen) | 0));
+        fn(frameLen, maxFramePayloadLen) >>> 0;
```

Comment:
The `>>> 0` isn't necessary, is it?  The assignment is going to coerce it to uint32 anyway.

Reply:
@bnoordhuis Yes. :) Updated!